### ISSUE TITLE
`npm run lint` failing on osx

### ISSUE THREAD

**Issue by splashcd** [coryhouse/javascript-development-environment#5]
I'm following along with the `Building a JavaScript Development Environment` Pluralsight course, Linting chapter - Demo video.  I just modified package.json and added .eslintrc.json

Running `npm run lint` produces an error, and creates the attached npm-debug.log file.

There are no spaces in any directories in my projects path.  Can you advise?

[npm-debug.txt](https://github.com/coryhouse/javascript-development-environment/files/784736/npm-debug.txt)

**Comment by coryhouse** [comment]
Does this issue occur if you download and run the final course exercise project?

Have you assured .eslintrc is in the root of your project? (same path as package.json)

**Comment by splashcd** [comment]
Yes, the issue does occur.  I've copied all files from your repo.  The .eslintrc.json and package.json are both in the root project directory.  The only thing I can see that may be different, is that I'm running in a virtualenv.

**Comment by coryhouse** [comment]
That's odd. I'm running the same versions on MacOS just fine. And unfortunately the npm-debug isn't providing me anything actionable. You might try reinstalling Node or installing the latest version of Node. 7.5 also runs fine on my Mac.

**Comment by splashcd** [comment]
Reinstalled, to no avail.  The issue stemmed from having the line 'debugger;'  in index.js

**Comment by cpey** [comment]
relevant thread: https://github.com/eslint/eslint/issues/7933
